hello_world/helpers.py

```diff
@@ -0,0 +1,130 @@
+from typing import Any, List, Optional, Tuple
+
+
+def parse_row(row: List[Any]) -> List[str]:
```

Comment:
Remember next time, we are using python3.9 which includes `list`, `tuple` as a typehint. But no problem, I am fine with anything.

```suggestion
def parse_row(row: list[Any]) -> list[str]:
```